tests/fixtures/feeds.json

```diff
@@ -24,12 +24,12 @@
         "args": {},
         "namespace": "{{ .Namespace }}"
       },
-      "source": {
+      "source": [{
         "config": {
           "series_id": "mosaic_series_id"
         },
         "type": "mosaic"
-      },
+      }],
```

Comment:
A feed.source is a list, see https://developers.planet.com/docs/analytics/reference/#tag/Feeds/paths/~1feeds~1{id}/get. By itself, the change to this file is enough to reproduce the error reported (offline).